src/v3/src/util/webauthnUtils.ts

```diff
@@ -71,10 +67,6 @@ export const webAuthNEnrollmentHandler: WebAuthNEnrollmentHandler = async (trans
  * Uses the Web Authentication API to retrieve credentials for a client to authenticate
  *
  * Used by {@link WebauthnControl.tsx} Renderer to initiate the WebAuthN Authentication flow.
- *
- * @param {IdxTransactionWithNextStep} transaction
- * @return {Promise<WebAuthNVerificationPayload>} ClientData, AuthenticatorData & SignatureData
```

Comment:
Any reason for removing this doc?

Reply:
Maybe I should not touch it in this PR, but eventually the transformer should handle transaction transformation but not just transforming the nextStep.